src/data/tickers.js

```diff
@@ -0,0 +1,602 @@
+export default [
```

Comment:
I would reduce this list to contains around 20 items

Reply:
why don't we just leave it as it is, and when we know exactly how much we want, we leave those? Having all of them is useful, for example made me realize that if we have too much we need to use a select with an autocomplete

Reply:
I think @mariocao said that we are going list 10 assets but leave it as it is until we have the list